components/augeas/augeas-lenses.p5m

```diff
@@ -0,0 +1,427 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-lenses@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) - Stock lenses"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+### In source dir do this:
+# find lenses -type f |  sed 's,^\(.*\)$,file \1 path=usr/share/augeas/\1,' >> ../augeas-lenses.p5m
+
+file lenses/ssh.aug path=usr/share/augeas/lenses/ssh.aug
```

Comment:
Perhaps, add $(SOURCE_DIR)/lenses to PKG_PROTO_DIRS and use transform to deliver just path actions? (see apr-util component for example) or some POST_INSTALL action to install them to PROTO area?